examples/session/nisyscfg.py

```diff
@@ -0,0 +1,158 @@
+r"""Get information about the remote machine using NI System Configuration.
+
+Getting Started:
+
+To run this example, install NI System Configuration on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.system-configuration.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Running from the command line:
+
+usage: nisyscfg.py [-h]
```

Comment:
I'm concerned about calling the example `nisyscfg`. It's not exposing the API. I'd rather keep distinct super-simple examples for enumerating devices and for enumerating software.

Reply:
> I'd rather keep distinct super-simple examples for enumerating devices and for enumerating software.

This conflicts with what @harsha-bhushan suggested above. I can either 
* Roll back to the two examples as I had previously, or 
* Rename the merged example here to not reference NISysCfg. I'm open to suggestions regarding the name. Maybe it could be called `server-resources` or something. Or 
* _\<some other suggestion here\>_

Reply:
I'm OK with a single example that doesn't claim to *be* nisyscfg. What about `enumerate-hardware-software.py`?

Reply:
That's fair. Though from what Harsha said (and others I've heard), it sounds like we may add more functions that call nisyscfg, so for the name, I was just going for something generic. I'm also using subcommands to determine what function is called/information is printed: in my current changes, users can do something like `python nisyscfg.py enumerate-devices` and it just prints out the list of devices. So `python enumerate-hardware-software.py enumerate-devices` feels a little redundant to me. More ideas for the name:  `session-utilities.py`, `server-info.py`

I'm not keen on making the example print out **both** a list of devices and software packages, in case that was being asked, because the list of software especially can get _very_ long (even leaving out hidden packages). But if subcommands aren't desirable either, I'm okay with going back to separate example modules.

Reply:
I don't have a strong opinion. Given the 2 names you suggested, I prefer `session-utilities.py`.